corehq/apps/reports/standard/monitoring.py

```diff
@@ -1524,9 +1526,17 @@ def _total_row(self, rows, report_data):
             total_row[3] = '%s / %s' % (len(active_users), len(all_users))
         else:
             num = len(filter(lambda row: row[3] != _(self.NO_FORMS_TEXT), rows))
+            case_owners = set()
+            for user in self.users_to_iterate:
+                case_owners = case_owners.union((user.user_id, user.location_id))
+                case_owners = case_owners.union(user.group_ids)
             total_row[3] = '%s / %s' % (num, len(rows))
-            total_row[6] = get_active_case_count(self.domain, self.datespan, self.case_types).total
-            total_row[7] = get_active_case_count(self.domain, self.datespan, self.case_types, True).total
+            total_row[6] = sum(
+                [int(get_active_case_counts_by_owner(self.domain, self.datespan, self.case_types).get(id, 0))
```

Comment:
Isn't this going to call `get_active_case_counts_by_owner` for each owner? Same for `get_total_case_counts_by_owner`.